Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Ride.hs

```diff
@@ -128,3 +132,67 @@ getRideStatus rideId personId = withLogTag ("personId-" <> personId.getId) do
         customer = SPerson.makePersonAPIEntity decRider,
         driverPosition = mbPos <&> (.currPoint)
       }
+
+mkCommonRideStatus :: RideStatus -> Common.RideStatus
+mkCommonRideStatus rs = case rs of
+  NEW -> Common.NEW
+  INPROGRESS -> Common.INPROGRESS
+  COMPLETED -> Common.COMPLETED
+  CANCELLED -> Common.CANCELLED
```

Comment:
In case we decide not to import this types from here to dashboard, we can also choose to use the existing `RideStatus `type of dashboard. It has a `RIDE_` enum as prefix of all these types. Although I'm not sure why it is made like that. 
cc : @roman-bodavskiy